[2023-10-17T11:19:05.149+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-17T11:19:05.156+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-17T11:19:05.156+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:05.168+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-27 00:00:00+00:00
[2023-10-17T11:19:05.174+0000] {standard_task_runner.py:57} INFO - Started process 540 to run task
[2023-10-17T11:19:05.177+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-27T00:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpo_mkq5j9']
[2023-10-17T11:19:05.178+0000] {standard_task_runner.py:85} INFO - Job 62: Subtask waiting_for_data
[2023-10-17T11:19:05.214+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:05.276+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-27T00:00:00+00:00'
[2023-10-17T11:19:05.284+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:05.284+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:05.284+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:05.284+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:05.292+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210227T000000, start_date=20231017T111905, end_date=20231017T111905
[2023-10-17T11:19:05.349+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:05.363+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:32.959+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-18T10:42:32.968+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-18T10:42:32.969+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:32.978+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-27 00:00:00+00:00
[2023-10-18T10:42:32.984+0000] {standard_task_runner.py:57} INFO - Started process 1502 to run task
[2023-10-18T10:42:32.987+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-27T00:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmph23nd7li']
[2023-10-18T10:42:32.988+0000] {standard_task_runner.py:85} INFO - Job 61: Subtask waiting_for_data
[2023-10-18T10:42:33.026+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:33.088+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-27T00:00:00+00:00'
[2023-10-18T10:42:33.106+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:33.109+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210227T000000, start_date=20231018T104232, end_date=20231018T104233
[2023-10-18T10:42:33.119+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 61 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1502)
[2023-10-18T10:42:33.159+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:33.194+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:04.437+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-18T11:17:04.449+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-18T11:17:04.450+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:04.468+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-27 00:00:00+00:00
[2023-10-18T11:17:04.477+0000] {standard_task_runner.py:57} INFO - Started process 278 to run task
[2023-10-18T11:17:04.482+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-27T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpdn3670go']
[2023-10-18T11:17:04.483+0000] {standard_task_runner.py:85} INFO - Job 78: Subtask waiting_for_data
[2023-10-18T11:17:04.540+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:04.616+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-27T00:00:00+00:00'
[2023-10-18T11:17:04.633+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:04.633+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:04.634+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:04.634+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:04.644+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210227T000000, start_date=20231018T111704, end_date=20231018T111704
[2023-10-18T11:17:04.699+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:04.722+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:40.858+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-19T12:11:40.880+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [queued]>
[2023-10-19T12:11:40.881+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:40.937+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-02-27 00:00:00+00:00
[2023-10-19T12:11:40.949+0000] {standard_task_runner.py:57} INFO - Started process 333 to run task
[2023-10-19T12:11:40.956+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-02-27T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpaxsyf53q']
[2023-10-19T12:11:40.958+0000] {standard_task_runner.py:85} INFO - Job 73: Subtask waiting_for_data
[2023-10-19T12:11:41.022+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-02-27T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:41.119+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-02-27T00:00:00+00:00'
[2023-10-19T12:11:41.134+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:41.135+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:41.135+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:41.135+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:41.147+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210227T000000, start_date=20231019T121140, end_date=20231019T121141
[2023-10-19T12:11:41.207+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:41.254+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
